src/br/com/ciandt/thiagodf/models/Comentario.java

```diff
@@ -0,0 +1,37 @@
+package br.com.ciandt.thiagodf.models;
+
+/**
+ * Classe de Comentário.
+ * 
+ * @author thiagodf
+ *
+ */
+public class Comentario {
+
+	private String comentario;
+	
+/**
```

Comment:
Identar todo este javadoc para deixar alinhado com o código conforme sugestão abaixo
```suggestion
      /**
       * Método Construtor para comentário.         
       */
	public Comentario() {}
```
Faça isso nos demais comentários dos métodos dentro da classe tbem

Reply:
Feito.